### PR TITLE
feat(api + cli): endpoint and command for changing freight aliases

### LINKED ISSUE #1074: add cli command for manually approving a piece of freight for a given stage
Follow up to #1073

### PR DESCRIPTION
**Pull request by krancour**
Fixes #1074 #1076

It made sense to tackle these together.

If you ignore codegen, this PR isn't nearly as big as it appears.

### CODE REVIEW

**Comment by krancour** [comment]
This needs some webhook modifications in case the alias is modified using `kubectl`.

**Review by rbreeze — APPROVED** [review]
LGTM after merge conflict resolution

**Pull request merged**
